src/detection/displayserver/linux/wmde.c

```diff
@@ -70,7 +70,8 @@ static void applyPrettyNameIfWM(FFDisplayServerResult* result, const char* proce
         strcasecmp(processName, "gnome-shell") == 0 ||
         strcasecmp(processName, "gnome shell") == 0 ||
         strcasecmp(processName, "gnome-session-binary") == 0 ||
-        strcasecmp(processName, "Mutter") == 0
+        strcasecmp(processName, "Mutter") == 0 ||
+        strcasecmp(processName, "Mutter (Muffin)") == 0
```

Comment:
This should go one below to the detection of `Muffin`, not `Mutter`.

On another note, where does that value come from? Doesn't seem to be either a process name nor an environment variable.

Reply:
I see... moved it down to Muffin detetection :+1: But to be honest: last time i dived into C is about 10 years ago :D However, i'm also pretty sure that this i not the process name, but i did not look where you set the processName var :)

Reply:
`processName` is a bit misleading to be honest. Originally i just went through all running processes and checked their names, thus the name. Now it can be either (in that order checked):
* The name the WM reports itself as when connecting to it
* A value set in one of the many standard env variables
* A process name of any running process

Maybe `Mutter (Muffin)` is coming from the first point. You can run `xprop -id "${"$(xprop -root -notype _NET_SUPPORTING_WM_CHECK)"##* }" -notype -len 100 -f _NET_WM_NAME 8t` to check for that.

Reply:
Great. I'll merge as soon as the pipeline succeeds.

Reply:
Just another side note: wmctrl returns also Mutter (Muffin)

```bash
❯ wmctrl -m
Name: Mutter (Muffin)
Class: N/A
PID: N/A
Window manager's "showing the desktop" mode: OFF
```